routes/api/v1/tasks.js

```diff
@@ -0,0 +1,61 @@
+const express = require("express");
+const router = express.Router();
+
+// CONTROLLERS
+
+const { catchErrors } = require("../../../config/errorHandler");
+const {
+	getTasks,
+	addTask,
+	updateTask,
+	getMyTasks,
+	deleteTask,
+	getTaskAssignees,
+	getUserTasks,
+	updateTaskAssignee
+} = require("../../../controllers/task_controller");
+const { allAuth, coreAuth } = require("../../../middlewares/auth");
+const {
+	addTaskValidation,
+	updateTaskValidation,
+	updateTaskAssigneeValidation
+} = require("../../../middlewares/validations");
+
+router.get(
+	"/assignees/:taid",
+	catchErrors(coreAuth),
```

Comment:
This won't a `member` who is head of a group to get the list of assignees.

Reply:
this is specifically for core/lead, the usual workflow won't require fetching taskAssignees  through an API, as they will be provided when fetching the task anyways.

only updating taskassignees is required as a separate route, which is open to members